### ISSUE TITLE
French Search screen bugs

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#311]
The following button labels are still read in English, after changing the language in French:

Search –  “Clear text” button should be labelled as “Effacez le texte” bouton
Search – Sort by options should be labelled as “trier par”

**Comment by AJBouzidi** [comment]
SearchScreen. line 434.

**Comment by ForumG-SC** [comment]
@AJBouzidi Hi Ali, this ticket was done , is the fix not working?

**Comment by AJBouzidi** [comment]
@ForumG-SC I believe this never been tested or not closed.

**Comment by ForumG-SC** [comment]
> @ForumG-SC I believe this never been tested or not closed.

This ticket can be tested in the current version and closed .

**Comment by AJBouzidi** [comment]
@ForumG-SC Could you please check "the focus will not go oldest to newest when using voiceover" before we close it. 
more often a ticket contains many issues. Bad practice should be avoided.

**Comment by ForumG-SC** [comment]
@AJBouzidi The focus is not going to oldest to newest as mentioned in the ticket's description , it goes in order newest to oldest first. See the attached screen recording video:

https://user-images.githubusercontent.com/84411852/167632793-d284ead5-1cfb-4b31-9d0b-097b31253d70.MOV

**Comment by zhuelle** [comment]
@ForumG-SC , should this ticket be assigned to business team as you moved it to Business to sign off?

**Comment by ForumG-SC** [comment]
@zhuelle I will remove it from the Business to sign off as this ticket was created by Rajat not business, it can be closed.

**Comment by Rajat-Sen** [comment]
When using French language, it says sort by instead of trier par.

Also, the focus should go like this: sort by > Newest to oldest > oldest to newest > filter by subject(s).

**Comment by giftydeep** [comment]
waiting on wifi to discuss this ticket in person with Jiandong.

**Comment by weglmar** [comment]
#474 
The search will be redesigned. This should be factored in when we are redesigning search.
